lib/helpers/mnemonic_encryptor.dart

```diff
@@ -0,0 +1,61 @@
+import 'dart:typed_data';
+
+import 'package:encrypt/encrypt.dart';
+import 'package:flutter/foundation.dart' as foundation;
+import 'package:flutter_secure_storage/flutter_secure_storage.dart';
+import 'package:shared_preferences/shared_preferences.dart';
+
+import '../global.dart';
+
+abstract class MnemonicEncryptor {
```

Comment:
we'll need to refactor this a bit in order to support testability and dependency injection, but that's after we come up with a common architecture for the mobile app